### PR TITLE
Fix Strategy page

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Fix migration Strategy page. To be merged after https://github.com/PrestaShop/docs/pull/1175

### CODE REVIEW

File: development/architecture/migration-guide/strategy.md [hunk 1]
```diff
@@ -117,15 +119,15 @@ These pages are form leveraging the concepts of CQRS.
 
 So "migrating a legacy Add/Edit page" means to "move the legacy logic into a Handler and building a Symfony Form on top of it with its management logic".
 
-# Form Theme
+## Form Theme
 
-We dont want to render manually Form inputs, so we now aim to build a Twig Form Theme allowing us to render all of our Symfony forms with a single block
+We dont want to render manually Form inputs, so we now rely on a [Twig Form Theme][twig-form-theme] allowing us to render all of our Symfony forms with a single block
```
Comment (eternoendless) [inline development/architecture/migration-guide/strategy.md]:
```suggestion
We don't want to render Form inputs manually, so we now rely on a [Twig Form Theme][twig-form-theme] allowing us to render all of our Symfony forms with a single block
```

**Review by atomiix — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @matks

**Comment by Progi1984** [comment]
Thanks @matks

**Comment by eternoendless** [comment]
This one doesn't need backport, right?

**Pull request merged**